README.md

```diff
@@ -9,41 +9,61 @@ So to launch the server, it will only start the server.
 Mods and bots are managed at startup by checking your `config.yml`.
 `npm` is only invoked if changes are made to your `config.yml`.
 
-## Usage
+## Installation (Recommended)
+**Step 1: Clone/Download this repository.**
```

Comment:
Maybe it would be better to just expose the `docker-compose.yml` and `config.yml` in a release.
But those are the only two files you need, so unsure if the best is to clone this repo. It is of course tiny.
I personally have a docker-compose and a config in my screeps bot repo : https://github.com/Jomik/screeps-ai

Reply:
We would need to expose the `.env` as well. I think this way is probably "easiest" per-se.